### ISSUE TITLE
Academic initiative enrolment needs a university account 

### ISSUE THREAD

**Issue by eslamkarim** [NorhanKhaled/SkillsAcademy-CAD#10]
to get the promo code we need to sign up for the initiative but it only accepts university accounts
also is a cloud lite account with a 256b machines will be enough for the courses?

![image](https://user-images.githubusercontent.com/26721393/80974596-373c8700-8e21-11ea-8a9b-d3c25547d9b3.png)

**Comment by kareem-zakaria** [comment]
![image](https://user-images.githubusercontent.com/50543444/80983814-4de8db00-8e2d-11ea-811a-f2e3cfd0c44e.png)

Same issue.

**Comment by aAbdelnaby88** [comment]
![Screenshot from 2020-05-04 17-49-29](https://user-images.githubusercontent.com/16451355/80985534-a5884600-8e2f-11ea-8e85-a8a89e63e9c6.png)

same issue too

**Comment by NorhanKhaled** [comment]
yes, It only accepts university accounts to be able to get the promocode that is needed by some exercises (eg. the Kubernetes).
if you have an IBM Cloud account with gmail, hotmail or yahoo . you won't be able to perform some exercises that requires extra features than a lite account

**Comment by eslamkarim** [comment]
So is there a solution or we just have to accept that we won't be able to do these labs?

**Comment by kareem-zakaria** [comment]
I solved the issue by using my university email instead, but I applied the promocode to the account with my other email, is that okay?

**Comment by atefhares** [comment]
@NorhanKhaled 

I have a university email and still not able to complete this step:
it syas your university not registered!

![Selection_048](https://user-images.githubusercontent.com/14354272/81117178-f6c73100-8f26-11ea-8004-474bb8b57eb6.png)


-------

Also I have an account on IBM cloud and I used a promo code from `cognitiveclass.ai` last year and subscription is finished, now whenever I log into my account it blocks me show this pop up 
and asks me to add a credit card! I can't even use the free tier! the only option available is to add a new promo code!

![Selection_049](https://user-images.githubusercontent.com/14354272/81117638-b7e5ab00-8f27-11ea-82cc-163b431a6996.png)


**Can I use another account on ibm cloud instead of the one registered at IBM skills academy?**

**Comment by NorhanKhaled** [comment]
@KareemAhmed96  yes it will be ok

**Comment by NorhanKhaled** [comment]
@atefhares please contact your university relation Leader for this issue 
and for the IBM Cloud account, yes you will need to use another one

**Comment by kareem-zakaria** [comment]
> @KareemAhmed96 yes it will be ok

@NorhanKhaled Thank you.
